lib/core/src/display/rendering/htmlrenderer.rs

```diff
@@ -0,0 +1,63 @@
+use super::Camera;
+use super::HTMLScene;
+use super::IntoCSSMatrix;
+use crate::system::web::StyleSetter;
+use crate::prelude::*;
+
+fn eps(value: f32) -> f32 {
+    if value.abs() < 1e-10 { 0.0 } else { value }
+}
+
+/// A renderer for `HTMLObject`s.
+#[derive(Default)]
+pub struct HTMLRenderer {}
+
+impl HTMLRenderer {
+    /// Creates a HTMLRenderer.
+    pub fn new() -> Self { default() }
+
+    /// Renders the `Scene` from `Camera`'s point of view.
+    pub fn render(&self, camera: &mut Camera, scene: &HTMLScene) {
+        let half_d = scene.get_dimensions() / 2.0;
+        // Note [znear from projection matrix]
+        let expr = camera.projection[(1, 1)]; // expr = 2.0 * near / (height)
```

Comment:
This hardcoded `(1,1)` should have a dedicated method or something. It's not descriptive enough what it really does.

Reply:
I really don't know how to simply describe it without explaining how a projection matrix is composed.